server/neptune/gateway/event/comment_handler.go

```diff
@@ -19,6 +20,8 @@ import (
 
 const warningMessage = "⚠️ WARNING ⚠️\n\n You are force applying changes from your PR instead of merging into your default branch 🚀. This can have unpredictable consequences 🙏🏽 and should only be used in an emergency 🆘.\n\n To confirm behavior, review and confirm the plan within the generated atlantis/deploy GH check below.\n\n 𝐓𝐡𝐢𝐬 𝐚𝐜𝐭𝐢𝐨𝐧 𝐰𝐢𝐥𝐥 𝐛𝐞 𝐚𝐮𝐝𝐢𝐭𝐞𝐝.\n"
 
+const legacyApplyCommandMessage = "⚠️ INFO ⚠️\n\n The apply command is no longer required! Simply merge your PR to apply your terraform changes in [Platform Mode](https://infradocs.lyft.net/deploy/infrastructure-as-code/platform_mode.html) "
```

Comment:
I don't think you need the ⚠️  symbols for an info message.  Also don't think you need the words info either imo